src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -144,6 +146,9 @@ export default class GroupComparisonPage extends React.Component<IGroupCompariso
                             <ProteinEnrichments store={this.store}/>
                         </MSKTab>
                     )}
+                    <MSKTab id={GroupComparisonTab.CLINICAL.toString()} linkText="Clinical">
```

Comment:
toString() unnecessary, since its a string-valued enum

Reply:
I thought we wanted this tab to be further to the left, next to survival or overlap? @jjgao